client/src/shared/components/form/ExportButton.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Labelled Input component
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { CSVLink } from 'react-csv';
+import { rgba } from 'polished';
+import { colors, fonts } from '../../style_guide';
+
+const Button = styled(CSVLink)`
```

Comment:
Styled components should be extended [like this](https://www.styled-components.com/docs/basics#extending-styles)

Reply:
It's not a styled component, the way it's done is correct.

Reply:
Oops ignore me 😊